README.md

```diff
@@ -426,6 +424,57 @@ __Example__
 var numbers = flyd.stream(0);
 var squaredNumbers = flyd.map(function(n) { return n*n; }, numbers);
 ```
+### flyd.chain(fn, s)
+Given a stream of streams, returns a single stream of merged values from the created streams.
```

Comment:
Isn't `s` here just `Stream a` rather than stream of streams?

Reply:
Yes, that's true, I copied this from that `flatMap` readme, I'll fix